queries/vue/injections.scm

```diff
@@ -2,7 +2,7 @@
   (style_element
     (start_tag) @_no_attribute
     (raw_text) @css)
-  (#match? @_no_attribute "<\\s*style\\s*>")
+  (#match? @_no_attribute "\\<\\s*style\\s*\\>")
```

Comment:
Somehow, without explicitly escaping `<` and `>`, the entire capture works incorrectly, unsure why that's the case